css/donate.css

```diff
@@ -28,52 +28,63 @@ h1 {
 
 h2 {
   text-align: center;
-  font-size: 40px;
+  font-size: 60px;
 }
 
 .curved {
   background-color: #ECE3F2;
   border-radius: 50% 50% 0 0 / 100% 100% 0 0;
-  padding-top: 70px;
+  padding-top: 50px;
 }
 
 .donate-header {
   padding-top: 50px;
   padding-bottom: 20px;
-  font-size:10px;
+  font-size: 10px;
 }
 
 .donate-story {
   background-color:#ECE3F2;
   padding-top: 20px;
```

Comment:
more context in the comment below, but can you get rid of the 20px padding and use
padding-top: 3%;
padding-bottom: 3%;
instead?